sprout/core/create_readme_text.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+
+def create_readme_text(properties: dict) -> str:
+    """Create a json object containing the readme text.
+
+    Args:
+      properties: An object containing the package and resource properties.
+
+    Returns:
+      A string for the README text
+    """
+    properties = json.loads(properties)
+    # TODO: Finish this once Properties have been finished.
+    # resources = # Python code to convert the resource details in the dict as
+    # a Markdown list.
+
+    readme_text = (
+        f"This is the README file for the package.\n\n"
+        f"{properties.name}: {properties.title}\n\n"
+        f"Within this data package contains:\n\n"
+        f"{properties.resources}"
```

Comment:
```suggestion
        f"{properties["name"]}: {properties["title"]}\n\n"
        f"Within this data package contains:\n\n"
        f"{properties["resources"]}"
```
Here we cannot access fields using a dot because `properties` is just a dictionary. If we passed `properties` as a `PackageProperties` we could use the dot notation. (Maybe we'll do that later, but I think for now everything else expects a `dict`.)